icu4j/maven-build/local-maven-repo/com/ibm/icu/icutzdata/73.1-SNAPSHOT/maven-metadata-local.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<metadata modelVersion="1.1.0">
+  <groupId>com.ibm.icu</groupId>
+  <artifactId>icutzdata</artifactId>
+  <version>73.1-SNAPSHOT</version>
+  <versioning>
+    <snapshot>
+      <localCopy>true</localCopy>
+    </snapshot>
+    <lastUpdated>20221119025532</lastUpdated>
+    <snapshotVersions>
+      <snapshotVersion>
+        <extension>jar</extension>
+        <value>73.1-SNAPSHOT</value>
+        <updated>20221119025532</updated>
```

Comment:
Are these files generated? docs?

Reply:
same. i don't think we should check in this local-maven-repo at all.  We might .gitignore it and then github action cache it.